### PR TITLE
A11y add aria expanded sub nav

### PR DESCRIPTION
**Pull request by DanielCliftonGuardian**
## What does this change?

This adds the aria-expanded attribute to the veggie burger "button" with dynamic true / false toggling.

This has been tested with VoiceOver.

This resolves https://github.com/guardian/dotcom-rendering/issues/5048 

## Why?

This was raised by the DAC report  https://drive.google.com/file/d/1hhrWd99v9tB7SNIDRWHy0svS3JyaKxeE/view

## Screenshots

image

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -24,5 +24,7 @@
     ],
     "git.branchProtection": [
         "main"
-    ]
+    ],
+    "javascript.preferences.importModuleSpecifier": "relative",
+    "typescript.preferences.importModuleSpecifier": "relative"
```
Comment (JamieB-gu) [inline .vscode/settings.json:29]: I'm not sure we want to enable this for all users? I can't speak for DCR, but AR generally doesn't use relative imports.

Comment (mxdvl) [reply .vscode/settings.json]: Oh, I thought this was true for the whole codebase, thanks for flagging, @JamieB-gu!

**Review by mxdvl — APPROVED** [review]
👏 

Excellent – it can be hard to test these scripts, well done!

**Review by OllysCoding — APPROVED** [review]
LGTM!

**Pull request merged**